packages/account/src/account.ts

```diff
@@ -632,6 +633,9 @@ export class Account extends AbstractAccount {
       );
     }
     const transactionRequest = transactionRequestify(transactionRequestLike);
+    if (isTransactionTypeUnknown(transactionRequest)) {
```

Comment:
Should this be warning before returning the transaction response?

Reply:
The [warning](https://github.com/FuelLabs/fuels-ts/pull/3003/files#diff-a7b58f093b62c1f41f0e8a5ea0700c9a0ba62d9f16a62c90e7e451cf30dba091R47-R49) is done in the `transactionRequestify` function which is what [this test](https://github.com/FuelLabs/fuels-ts/pull/3003/files#diff-3ccf1abbc4dc63f3169e41bf35f6a65f268903ebf6185701aff6ec308e975ce7R25-R54) verifies.